src/Mono.Android.Runtime/Mono.Android.Runtime.csproj

```diff
@@ -31,11 +31,11 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
-  <!-- Only build the .NET 6+ version of 'Mono.Android.Export.dll' for the latest stable Android version. -->
```

Comment:
@pjcollins Can you advise here? These lines seem to have been copied from `Mono.Android.Export.csproj`, but I think they are needed in `Export` because it is built for different API levels:

https://github.com/xamarin/xamarin-android/blob/main/build-tools/xaprepare/xaprepare/Application/GeneratedMakeRulesFile.cs#L150-L153

What needs to happen for `Mono.Android.Runtime` which is a "normal" project?

Reply:
I don't think this needs to change unless it is causing an issue. I believe this was added because `Mono.Android.csproj` references `Mono.Android.Runtime.csproj`, and we don't want/need to rebuild this project every time `Mono.Android.csproj` is built.

Reply:
It did cause an issue when building the the unstable API-34 `Mono.Android.dll`:

```
c:\code\xamarin-android\src\Mono.Android\Android.Runtime\AndroidRuntime.cs(20,41): error CS0103: The name 'RuntimeConstants' does not exist in the current context [c:\code\xamarin-android\src\Mono.Android\Mono.Android.csproj::TargetFramework=net8.0]
c:\code\xamarin-android\src\Mono.Android\Android.Runtime\Logger.cs(47,43): error CS0246: The type or namespace name 'LogLevel' could not be found (are you missing a using directive or an assembly reference?) [c:\code\xamarin-android\src\Mono.Android\Mono.Android.csproj::TargetFramework=net8.0]
c:\code\xamarin-android\src\Mono.Android\Android.Runtime\Logger.cs(49,27): error CS0246: The type or namespace name 'LogLevel' could not be found (are you missing a using directive or an assembly reference?) [c:\code\xamarin-android\src\Mono.Android\Mono.Android.csproj::TargetFramework=net8.0]
c:\code\xamarin-android\src\Mono.Android\Android.Runtime\Logger.cs(8,19): error CS0246: The type or namespace name 'LogCategories' could not be found (are you missing a using directive or an assembly reference?) [c:\code\xamarin-android\src\Mono.Android\Mono.Android.csproj::TargetFramework=net8.0]
c:\code\xamarin-android\src\Mono.Android\Android.Runtime\Logger.cs(63,15): error CS0103: The name 'RuntimeConstants' does not exist in the current context [c:\code\xamarin-android\src\Mono.Android\Mono.Android.csproj::TargetFramework=net8.0]
```

([Source](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7332897&view=results))

This seemed to be caused by the `Runtime` being missing due to the lack of compilation.

Commenting this optimization out fixes it, but I'm not sure if that's the solution we want.

Reply:
Oh I see this file was added somewhat recently, in that case I think a better fix would be to use the same conditions defined in [Mono.Android.csproj](https://github.com/xamarin/xamarin-android/blob/main/src/Mono.Android/Mono.Android.csproj#L411-L420):

```xml
<!-- Only build the .NET 6+ version of 'Mono.Android.dll' for the default API level that is supported or higher. -->
  <PropertyGroup Condition=" '$(TargetFramework)' != 'monoandroid10' And '$(AndroidApiLevel)' &lt; '$(AndroidDefaultTargetDotnetApiLevel)' ">
    <BuildDependsOn></BuildDependsOn>
  </PropertyGroup>

  <PropertyGroup Condition=" '$(TargetFramework)' != 'monoandroid10' And '$(AndroidApiLevel)' &gt;= '$(AndroidDefaultTargetDotnetApiLevel)' ">
    <BuildDependsOn>
      $(BuildDependsOn);
      _CopyToPackDirs;
    </BuildDependsOn>
  </PropertyGroup>
```

Reply:
Cool, that seems to work.